### ISSUE TITLE
Button Link in Self-Checkin email is broken

### ISSUE THREAD

**Issue by blakenan-bellese** [CodeForBaltimore/Healthcare-Rollcall#225]
**Describe the bug**
The system does not display the self check in page when the user clicks on the check in button embedded in an email.

**To Reproduce**
1. Send an email to yourself
2. Click on button in the checkin email
3. You are NOT taken to a self checkin page with a questionnaire 

**Expected behavior**
The button on the email should taker the user to a questionnaire page to complete a self checkin.

**Environment**
Happens in both local install and on PROD.  If my memory is accurate both seems to be flaws in the domain portion of URL is either missing or wrong.

**Comment by blakenan-bellese** [comment]
![Screen Shot 2020-12-08 at 8 55 58 AM](https://user-images.githubusercontent.com/61432973/101492859-42194600-3933-11eb-9fc6-233e7b9d4698.png)
Here's a screen shot of what the browser displays after clicking on the check in button.

**Comment by blakenan-bellese** [comment]
Did some snooping and, while I'm no expert, seems to me that the issue is due to a lack of (or bad value of) environment variable "URL" which is referenced in line 60 of `/src/email/index.js` but does not seem to be set anywhere nor is it included in the documentation as a necessary variable defined in `.env`

**Comment by ijadams** [comment]
I can take a look

**Comment by blakenan-bellese** [comment]
@ijadams Thanks.  One bit of info that may, or may not, interest you...  In order to get my local env to work with the MailDev SMTP server and avoid certificate issues I had to set `NODE_TLS_REJECT_UNAUTHORIZED='0'`.  The side affect of this was that when I added `URL` variable (set to `localhost:8080`) to my `.env` file, the back end thre the following erre when I tried to access the self checkin:
`(node:74204) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.`

I manually edited the URL of the checkin manually in my bowser to lead with http:// and I was able to access the self checkin.
